script.service.argonforty-device/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.service.argonforty-device" name="ArgonForty Device Configuration" version="0.0.1" provider-name="ArgonForty">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="virtual.rpi-tools" />

```

Comment:
this seems to be specific to libreleec and for a specific platform. Not sure our repo is the place for this...
@lrusak @chewitt can you guys help out?

Reply:
@ArgonForty If the goal is to add to support into LE specifically our own repo would be more appropriate. If the goal is to support multiple pi-oriented distros (LE, OSMC, Kodi on Rasperry Pi OS, etc.) then you can submit here, but you will need to provide any Python dependencies from the Kodi repo too - e.g. an OSMC user will not find rpi-tools because it only exists in LE.

Reply:
Hi @chewitt how do we submit to the LE repo? Thanks in advance.

Reply:
@chewitt Thank you for looking into this. We are actually looking to add this add on to support LE specifically. But we cannot find any information online how to submit addons for LE. Not sure if you can help point how we can submit this for LE ?

Reply:
@enen92 thank you for the help.